src/backend/distributed/commands/multi_copy.c

```diff
@@ -137,24 +136,28 @@ static PGconn *masterConnection = NULL;
 /* Local functions forward declarations */
 static void CopyFromWorkerNode(CopyStmt *copyStatement, char *completionTag);
 static void CopyToExistingShards(CopyStmt *copyStatement, char *completionTag);
-static void CopyToNewShards(CopyStmt *copyStatement, char *completionTag, Oid relationId);
+static void CopyToNewShards(CopyStmt *copyStatement, char *completionTag,
+							Oid relationId);
 static char MasterPartitionMethod(RangeVar *relation);
 static void RemoveMasterOptions(CopyStmt *copyStatement);
 static void OpenCopyTransactions(CopyStmt *copyStatement,
-								 ShardConnections *shardConnections, bool stopOnFailure);
+								 ShardConnections *shardConnections, bool stopOnFailure,
+								 bool binaryCopyFormat);
+static bool BinaryCopyFormat(TupleDesc tupleDescription, CopyOutState rowOutputState);
```

Comment:
I think this should be something like `RequiresBinaryCopyFormat` or `ShouldUseBinaryCopyFormat` to make it clear it returns a `bool`.